### PR TITLE
feat(wallets): :sparkles: add UI elements for XRP integration

### PR DESCRIPTION
**Pull request by heorhi-deriv**
## Changes:

- add XRP currency icon
- add XRP color background for wallet card and cashier overlay header

### Screenshots:

Screenshot 2024-07-22 at 15 37 31
Screenshot 2024-07-22 at 15 37 39
Screenshot 2024-07-22 at 15 37 48

### CODE REVIEW

File: packages/core/src/sass/app/_common/components/wallet.scss [hunk 1]
```diff
@@ -707,6 +707,44 @@ $wallet-bg-color: (
             ),
         ),
     ),
+    'xrp': (
```
Comment (nijil-deriv) [inline packages/core/src/sass/app/_common/components/wallet.scss:710]: Do we use the `wallet-bg-color` mixin anywhere?

**Review by aizad-deriv — APPROVED** [review]

File: packages/components/src/components/icon/icons.js [hunk 2]
```diff
@@ -1082,3 +1082,5 @@ import './wallet/ic-wallet-upgrade-completed.svg';
 import './wallet/ic-wallet-upgrade-logout.svg';
 import './wallet/ic-wallet-usd-coin-dark.svg';
 import './wallet/ic-wallet-usd-coin-light.svg';
+import './wallet/ic-wallet-xrp-dark.svg';
+import './wallet/ic-wallet-xrp-light.svg';
```
Comment (aizad-deriv) [inline packages/components/src/components/icon/icons.js:1086]: There is XRP icon in quill icons:
`PaymentMethodXrpBrandIcon`
`PaymentMethodXrpWhiteIcon `

Comment (heorhi-deriv) [reply packages/components/src/components/icon/icons.js:1086]: yes @aizad-deriv 
I'm using XRP currency icons from quill in `packages/wallets/src/constants/icons.ts`. 
The other place where we are using icons `as name` is in `useStoreWalletAccountsList`. That's why i added svg icons. In order to replace them with quill icons we need to refactor the hook. For now i didn't do that.

File: packages/core/src/sass/app/_common/components/wallet.scss (hunk 1 above)
Comment (heorhi-deriv) [reply packages/core/src/sass/app/_common/components/wallet.scss:710]: yes @nijil-deriv 
Here we are using wallet-bg-color for wallet icon
Screenshot 2024-07-22 at 15 37 58

File: packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss [hunk 3]
```diff
@@ -803,6 +803,56 @@
             }
         }
     }
+
+    &--XRP {
```
[the remark this answers is not in the document]
Comment (heorhi-deriv) [reply packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss:807]: mmm yes @wojciech-deriv 
this file needs refactoring actually

**Review by aizad-deriv — APPROVED** [review]

**Review by jim-deriv — APPROVED** [review]

**Pull request merged**